lib/upgrade/etcd/conversionv1v3/ippool_test.go

```diff
@@ -154,7 +154,7 @@ var poolTable = []struct {
 				},
 				Spec: apiv3.IPPoolSpec{
 					CIDR:        "6.6.6.6/26",
-					IPIPMode:    apiv3.IPIPModeNever,
+					IPIPMode:    apiv3.IPIPModeAlways,
```

Comment:
Nope still wrong.

The IPIP interface field is what determines whether IPIP is enabled or not.

```
IPIP interface  IPIPMode        v3 IPIPMode
not set              -              Never
set                    not set          Always
set                    cross-subnet    CrossSubnet
set                    not cross-subnet      Always
```

Reply:
i've added what i think is the correct logic above.